assets/js/lsdoc/comments.js

```diff
@@ -1,24 +1,25 @@
-var CommentSystem = (function(){
+var CommentSystem = (function (){
     var commentItem = {
         itemId: null,
         itemType: 'document'
     };
 
-    function exportCSV(){
+    function exportCSV (){
         var url = "/salt/case/export_comment/"+commentItem.itemType+"/"+commentItem.itemId+"/comment.csv";
         window.location = url;
     }
 
-    function init(nodeRef){
+    function init (nodeRef){
         setItem(nodeRef);
         $('.js-comments-container').comments({
             profilePictureUrl: '',
             enableDeletingCommentWithReplies: true,
-            getComments: function(success, error) {
+            enableAttachments: true,
```

Comment:
I thought attachments is supposed to be an option, so a way to disable it is necessary.

Reply:
you're right